src/cloudWatchLogs/commands/searchLogGroup.ts

```diff
@@ -8,63 +8,96 @@ import * as telemetry from '../../shared/telemetry/telemetry'
 import {
     CloudWatchLogsData,
     CloudWatchLogsGroupInfo,
-    CloudWatchLogsParameters,
     LogStreamRegistry,
     filterLogEventsFromUriComponents,
 } from '../registry/logStreamRegistry'
-import { createQuickPick, DataQuickPickItem } from '../../shared/ui/pickerPrompter'
+import { DataQuickPickItem } from '../../shared/ui/pickerPrompter'
 import { Wizard } from '../../shared/wizards/wizard'
 import { createInputBox } from '../../shared/ui/inputPrompter'
 import { createURIFromArgs } from '../cloudWatchLogsUtils'
 import { DefaultCloudWatchLogsClient } from '../../shared/clients/cloudWatchLogsClient'
 import { CloudWatchLogs } from 'aws-sdk'
+import { LogGroupNode } from '../explorer/logGroupNode'
+import { RegionSubmenu, RegionSubmenuResponse } from '../../shared/ui/common/regionSubmenu'
 import { highlightDocument } from '../document/logStreamDocumentProvider'
 
-export async function searchLogGroup(registry: LogStreamRegistry): Promise<void> {
-    let result: telemetry.Result = 'Succeeded'
-    const regionCode = 'us-west-2'
-    const client = new DefaultCloudWatchLogsClient(regionCode)
-    const logGroups = await logGroupsToArray(client.describeLogGroups())
-    const response = await new SearchLogGroupWizard(logGroups).run()
-    if (response) {
-        const logGroupInfo: CloudWatchLogsGroupInfo = {
-            groupName: response.logGroup,
-            regionName: regionCode,
-        }
+function handleWizardResponse(response: SearchLogGroupWizardResponse, registry: LogStreamRegistry) {
```

Comment:
It's kind of implicit but
```suggestion
function handleWizardResponse(response: SearchLogGroupWizardResponse, registry: LogStreamRegistry): CloudWatchLogsData {
```